zsl/interface/webservice/utils/response_headers.py

```diff
@@ -20,13 +20,14 @@ def append_crossdomain(response, config=Injected):
 
     if config.get('ALLOW_ORIGIN'):
         response.headers['Access-Control-Allow-Origin'] = config.get('ALLOW_ORIGIN')
+
     response.headers['Access-Control-Allow-Methods'] = 'POST, GET, OPTIONS, PUT, DELETE'
-    response.headers['Access-Control-Allow-Headers'] = 'accept, origin, content-type'
+    response.headers['Access-Control-Allow-Headers'] = 'accept, origin, content-type, location, X-Total-Count, Link'
```

Comment:
I don't know if the other headers should be added for all queries, but for now they are.